cli

```diff
@@ -104,6 +104,42 @@ parser_list_expired_accounts = subparsers.add_parser(
     help="List users that have to be deactivated",
 )
 
+## List Last Logons for Users
```

Comment:
not necessarily needed for this PR, but this thing is complicated enough now that the main flow should probably be wrapped in a `main()` function

Reply:
Added this with a recent commit